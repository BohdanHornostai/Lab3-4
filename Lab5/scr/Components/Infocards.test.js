import React from 'react'
import {render, cleanup} from '@testing-library/react'
import Infocards from './Infocards'

 afterEach(cleanup)
 
 it('should take a snapshot', () => {
    const { asFragment } = render(<Infocards />)

    expect(asFragment(<Infocards />)).toMatchSnapshot()
   })