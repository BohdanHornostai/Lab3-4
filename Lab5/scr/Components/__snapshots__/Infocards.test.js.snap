// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="content"
    >
      <div
        class="card"
      >
        <div
          class="icon"
        >
          <i
            class="material-icons md-36"
          >
            attach_money
          </i>
        </div>
        <p
          class="title"
        >
          Sell
        </p>
        <a
          class="text"
          href="#"
        >
          Make your own product
        </a>
      </div>
      <div
        class="card"
      >
        <div
          class="icon"
        >
          <i
            class="material-icons md-36"
          >
            storefront
          </i>
        </div>
        <p
          class="title"
        >
          Buy
        </p>
        <a
          class="text"
          href="/sellers"
        >
          Find any product
        </a>
      </div>
      <div
        class="card"
      >
        <div
          class="icon"
        >
          <i
            class="material-icons md-36"
          >
            whatshot
          </i>
        </div>
        <p
          class="title"
        >
          Create
        </p>
        <a
          class="text"
          href="#"
        >
          Become the top seller
        </a>
      </div>
      <div
        class="card"
      >
        <div
          class="icon"
        >
          <i
            class="material-icons md-36"
          >
            explore
          </i>
        </div>
        <p
          class="title"
        >
          Explore
        </p>
        <a
          class="text"
          href="/sellers"
        >
          Explore the sea of products
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
